libs/payments/ui/src/lib/utils/types.ts

```diff
@@ -0,0 +1,6 @@
+export enum SupportedPages {
```

Comment:
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
* file, You can obtain one at http://mozilla.org/MPL/2.0/. */